src/components/AIBox.tsx

```diff
@@ -0,0 +1,67 @@
+import useAIComm from "~/hooks/useAIComm";
+
+const AIBox = ({
+  sessionId,
+  userId,
+  userName,
+  className,
+}: {
+  sessionId: string;
+  userId: string;
+  userName: string;
+  className?: string | undefined;
```

Comment:
```suggestion
  className?: string;
```
hi kf, iinw typescript will generate an `| undefined` to the type when it sees that the prop is optional `?`